python_modules/dagster-test/dagster_test/toys/alert_helpers.py

```diff
@@ -0,0 +1,44 @@
+import random
+import time
+
+import dagster as dg
+
+"""
+Repo of assets and ops that are useful for testing alerts
+"""
+
+
+def generate_random_schema(column_names: list[str]):
+    column_types = ["string", "int", "timestamp", "float"]
+    tags = [{"priority": "high"}, {"team": "marketing"}, {"region": "eu"}]
+    columns = [
+        dg.TableColumn(
+            name=column_name,
+            type=random.choice(column_types),
+            tags=random.choice(tags),
+            constraints=dg.TableColumnConstraints(
+                nullable=random.choice([True, False]), unique=random.choice([True, False])
+            ),
+        )
+        for column_name in column_names
+    ]
+
+    return dg.TableSchema(columns=columns)
+
+
+@dg.asset
```

Comment:
optional config that lets you specify a particular type or something could also be useful